packages/ra-ui-materialui/src/field/ReferenceField.js

```diff
@@ -144,7 +144,11 @@ export const ReferenceFieldView = ({
     }
     if (error) {
         return (
-            <Error aria-errormessage="Error" color="error" fontSize="small" />
+            <Error
+                aria-errormessage={error.message ? error.message : error}
```

Comment:
Shouldn't we translate this error ?

Reply:
Ideally, yes. In a future PR? I'd like to have this pr merged quickly to release a new alpha soon.